### ISSUE TITLE
Image direction

### ISSUE THREAD

**Issue by kazuyakusastro** [malikolivier/aflak#8]
The image in aflak is misaligned. It should be rotated by 90 degrees CCW. I may have told wrong directions for fits headers "CD1_1" and so on.

**Comment by malikolivier** [comment]
Excuse me for my lack of knowledge, but what is the criterion for the correct orientation of the image?

Currently, `aflak' displays the raw image as is. It does not rotate it. So I doubt the `CD1_1` header is at fault. I think we should take into account these headers to show the correct orientation. Is my assumption correct?

By the way, the N (red)/E (blue) compass should be correct. Can you check?

**Comment by kazuyakusastro** [comment]
I compared the aflak image with SDSS image on the website.
http://skyserver.sdss.org/dr14/en/tools/explore/Summary.aspx?
I checked an object, SDSS J151806.13+424445.0. (The default object of this website?) The SDSS images are aligned properly. A round tidal tail appears in NW of the brightest core, the 2nd brightest core and a straight tail in S-SE. Then, I think the aflak image is rotated by 90 degrees CCW.

The compass is rotated 90 degrees with respect to the aflak image. Please rotate not the compass but the image, because the North is usually up and east is left.

**Comment by malikolivier** [comment]
> The  compass is rotated 90 degrees with respect to the aflak image. Please  rotate not the compass but the image, because the North is usually up  and east is left.

Thanks. The direction of the compass is directly computed from the content of the FITS header, so I think it is indeed correct.

> I think the aflak image is rotated by 90 degrees CCW.

This is `manga-7443-12703-LINCUBE.fits` after I rotated it:

![image](https://user-images.githubusercontent.com/14120117/35506848-190bf032-052e-11e8-9642-e45e2f067a0d.png)

Does that look OK?

The code has the diff below. As you can see, I am just rotating the raw data in the image, irrespective of the FITS header.

```diff
diff --git a/aflak/fits.py b/aflak/fits.py
index b78703d..c6dc216 100644
--- a/aflak/fits.py
+++ b/aflak/fits.py
@@ -21,7 +21,7 @@ class FITS:
         return self.hdulist['WAVE'].data
 
     def flux(self):
-        return self.hdulist['FLUX'].data
+        return np.rot90(self.hdulist['FLUX'].data.swapaxes(0, 2)).swapaxes(0, 2)
 
     def wave_unit(self):
         """
```

**Comment by kazuyakusastro** [comment]
Great! The image is aligned properly in this figure.

I'd like to add one more thing. The Y-axis value, declination, should increase in the North. The X-axis value, right ascension, is collect; right ascension increases in the East.

**Comment by malikolivier** [comment]
I pushed a fix in 240e4cf , so I close this issue.
